source/endpoints/devices/Devices.py

```diff
@@ -0,0 +1,16 @@
+from flask_restplus import Resource
+
+from endpoints.devices import DAO, ns
+from endpoints.devices.Device import device
+
+
+@ns.route('/')
+class Devices(Resource):
+    '''Shows a list of all devices, and lets you POST to add new device'''
+    @ns.doc('list_devices')
+    @ns.marshal_list_with(device)
+    def get(self):
+        '''List all devices'''
+        return DAO.get()
```

Comment:
Is get() clear enough? We could make it getDevices(). This will also make it easier in the future when the database contains more than just the devices